src/components/explore/ComparisonSummary.svelte

```diff
@@ -135,7 +136,11 @@
     Summary
     <span
       use:tooltipAction={{
-        text: 'Compares the numeric values of the reference ⭑ to the hovered values ●',
+        text: `Compares the numeric values of the reference ⭑ to the hovered values ●. ${
+          justOne
+            ? `Please note that currently this probe doesn't have enough data to produce a meaningful comparison yet.`
```

Comment:
This is still confusing to me. While we say there is not enough data to produce meaningful comparisons yet, we still show _something_ on the graph. This might just be my lack of GLAM understanding showing, but thought it would be worth pointing out.